fastlane/Fastfile

```diff
@@ -300,32 +300,49 @@ platform :android do
   # Update the rollout of all 3 variants (app, automotive, wear) of the current version to the given value
   #
   # @param percent [Float] The rollout percentage, between 0 and 1
+  # @param track [String]
+  #        If `beta`, will update the rollout of the latest build in all 3 Beta tracks.
+  #        If `production`, will update the rollout of the latest build in all 3 Production tracks.
+  #        If `nil`, will derive the track based on if the `version_name_current` contains `-rc-` or not.
```

Comment:
I wonder if it wouldn't be safer to require `track` parameter. We don't update rollout often so it's not a big hindrance to type couple of letters more and it would prevent us from making a mistake either during manual deployment or when creating scripts. WDYT?